lib/Generator.php

```diff
@@ -52,6 +53,41 @@ public function __construct(PasswordPolicyConfig $config,
 	 * @throws HintException
 	 */
 	public function generate(): string {
+		if ($this->config->getGeneratePassphrases()) {
+			return $this->generate_phrase();
+		} else {
+			return $this->generate_random();
+		}
+	}
+
+	private function generate_phrase(): string {
+		$minLength = max($this->config->getMinLength(), 8);
+		$word_count = max($this->config->getWordCount(), 3);
```

Comment:
Follow the pattern with camelCase with lower case first as var name.
```suggestion
		$wordCount = max($this->config->getWordCount(), 3);
```